cleo_plugins/DebugUtils/SA.DebugUtils.ini

```diff
@@ -23,6 +23,6 @@ ColorError = "FF30EEFF"
 ColorDebug = "FFEE30FF"
 ColorSystem = "DDDDDDFF"
 
-; limits between 'wait' commands in script after which it is considered hanging
+; limits peer render frame after which script is considered hanging
```

Comment:
; maximum instructions per script per frame. Set to 0 to disable the limit.

Reply:
Added

Reply:
does the parser support inline comments?

can you delete the comment above?

Reply:
While parsing numbers INI related APIs just stop after first non numeric character.